docsteady/vcd.py

```diff
@@ -461,19 +461,17 @@ def summary(dictionary, comp, user, passwd):
     ve_status.append(['-1', len(veduplicated), "Duplicated"])
 
     # get VE real coverage
-    cover_names = ['No Test Cases Related', 'No Test Cases Executed', 'Test Cases Partially Executed',
-                   'Some Test Cases Fails', 'All Test Cases Pass', 'All Test Cases Fails']
+    # for reference: cover_names = ['No Test Cases Related', 'No Test Cases Executed', 'Test Cases Partially Executed',
+    #               'Some Test Cases Fails', 'All Test Cases Pass', 'All Test Cases Fails']
     vecoverage = [0, 0, 0, 0]
```

Comment:
It would be much better to make this (and similar variables below) something more descriptive: an actual class, a `dataclass`, or at worst a dictionary.  Having positional access with no description (and the `cover_names` can't be a description since there are more of them than there are array elements) is error-prone and not understandable to the reader.